src/containers/Header/Header.js

```diff
@@ -1,54 +1,55 @@
-import React from 'react'

```

Comment:
could not check this pull request due to it has error

![css-modules-errors](https://user-images.githubusercontent.com/6079597/133386455-d203035b-4332-424d-9206-5dc0d372f7ff.png)